apps/parsley/src/utils/searchLogs/index.ts

```diff
@@ -59,7 +60,7 @@ const searchLogs = (options: searchOptions): number[] => {
       }
     } else if (isLogRow(processedLogLine)) {
       if (processedLogLine >= lowerBound) {
-        if (searchRegex.test(getLine(processedLogLine))) {
+        if (searchRegex.test(stripAnsi(getLine(processedLogLine)))) {
```

Comment:
i think you also have to add this on Line 57; for the case that all sections and all subsections are collapsed, it won't detect a match

<img width="1081" alt="Screenshot 2025-06-24 at 2 24 07 PM" src="https://github.com/user-attachments/assets/32b4b6d1-e81c-420b-98b8-119edc87d40b" />